src/core/Akka/IO/UdpConnected.cs

```diff
@@ -433,23 +433,17 @@ private void OnComplete(object sender, SocketAsyncEventArgs e)
         {
             var actorRef = e.UserToken as IActorRef;
             actorRef?.Tell(ResolveMessage(e));
+            SocketEventArgsPool.Release(e);
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        private UdpConnected.SocketCompleted ResolveMessage(SocketAsyncEventArgs e)
+        private static UdpConnected.SocketCompleted ResolveMessage(SocketAsyncEventArgs e)
         {
             switch (e.LastOperation)
             {
                 case SocketAsyncOperation.Receive:
                 case SocketAsyncOperation.ReceiveFrom:
                     return new UdpConnected.SocketReceived(e);
-                case SocketAsyncOperation.Send:
```

Comment:
Same reason as above, this isn't being used.

Reply:
Got it